src/Symfony/Component/Serializer/Normalizer/GetSetMethodNormalizer.php

```diff
@@ -110,7 +110,7 @@ protected function extractAttributes($object, $format = null, array $context = [
 
             $attributeName = lcfirst(substr($method->name, 0 === strpos($method->name, 'is') ? 2 : 3));
 
-            if ($this->isAllowedAttribute($object, $attributeName)) {
+            if ($this->isAllowedAttribute($object, $attributeName, null, $context)) {
```

Comment:
Why don't we forward `$format` here as the third argument?

Reply:
@xabbuh You're right. Fixed !